msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/mwpj/TestCase2579654.kt

```diff
@@ -53,6 +54,7 @@ import org.junit.rules.TestRule
 // [MWPJ] After entry migration PRT is still usable without extra prompts.
 @LocalBrokerHostDebugUiTest
 @SupportedBrokers(brokers = [BrokerHost::class])
+@RetryOnFailure
```

Comment:
no limit?

Reply:
Not giving a number means it will retry one time